notify/email/email.go

```diff
@@ -236,6 +243,13 @@ func (n *Email) Notify(ctx context.Context, as ...*types.Alert) (bool, error) {
 		fmt.Fprintf(buffer, "%s: %s\r\n", header, mime.QEncoding.Encode("utf-8", value))
 	}
 
+	if reHeaderMessageId.Find(buffer.Bytes()) == nil {
```

Comment:
I'd rather check that `n.conf.Headers` doesn't contain the `message-id` key.

Reply:
done